docs/Reference/API-Methods.md

```diff
@@ -7243,10 +7248,109 @@ one object per call, in transaction execution order
         },
         ```
 
+### `trace_callMany`
+
+Performs multiple call traces on top of the same block. You can trace dependent transactions.
+
+!!! important
+
+    The requested block must be within [the number of pruning blocks retained](../CLI/CLI-Syntax#pruning-blocks-retained)
+    (by default, 1024).
+
+#### Parameters
+
+* `options`: *array* of *strings* - list of tracing options; tracing options are
+  [`trace`, `vmTrace`, and `stateDiff`](Trace-Types.md). Specify any
+  combination of the three options including none of them.
+
+* `blockNumber`: *string* - integer representing a block number or one of the string tags `latest`,
+  `earliest`, or `pending`, as described in
+  [Block Parameter](../HowTo/Interact/APIs/Using-JSON-RPC-API.md#block-parameter)
+
+#### Returns
+
+`result`: *array* of *objects* - list of [calls to other contracts](Trace-Types.md#trace) containing
+one object per call, in transaction execution order
+
+!!! example
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -X POST --data '{"jsonrpc":"2.0","method":"trace_callMany","params":[[[{"from":"0x407d73d8a49eeb85d32cf465507dd71d507100c1","to":"0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b","value":"0x186a0"},["trace"]],[{"from":"0x407d73d8a49eeb85d32cf465507dd71d507100c1","to":"0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b","value":"0x186a0"},["trace"]]],"latest"],"id":1}' http://127.0.0.1:8545
```

Comment:
In the trace call example request, we're missing a comma between the call object and the ['trace'] array it seems, can we fix it as well (this is a good example)